src/lib/fauna-account-client.mjs

```diff
@@ -204,12 +207,50 @@ export class FaunaAccountClient {
       method: "POST",
       path: "/databases/keys",
       body: JSON.stringify({
-        path,
         role,
+        path: this.standardizeRegion(path),
         ttl: ttl || new Date(Date.now() + TTL_DEFAULT_MS).toISOString(),
         name: "System generated shell key",
       }),
       secret: this.accountKeys.key,
     });
   }
+
+  /**
+   * Transforms database paths to standardize region group naming conventions expected by
+   * the account API.
+   *
+   * @param {string} databasePath - The database path to standardize
+   * @returns {string} The standardized path
+   * @example
+   * // Returns "us-std/my-database"
+   * standardizeRegion("us/my-database")
+   *
+   * // Returns "eu-std/my-database"
+   * standardizeRegion("eu/my-database")
+   *
+   * // Returns "global/my-database"
+   * standardizeRegion("classic/my-database")
+   *
+   * @throws {TypeError} If databasePath is provided but not a string
+   */
+  standardizeRegion(databasePath) {
+    if (!databasePath) return databasePath;
```

Comment:
what case is this handling & should we prefer to throw instead?

Reply:
I don't think so - an undefined or empty path is valid in some cases. For instance on the list database API.